REQUIRE

```diff
@@ -1,5 +1,4 @@
 julia 0.4
-FactCheck 0.3
```

Comment:
Don't we need this here, to point out that the package uses this? I see that it appears in `test/REQUIRE`, but isn't it necessary here to install it as a dependency when using `Pkg.add("ValidatedNumerics")`?